### ISSUE TITLE
stockfish 6

### ISSUE THREAD

**Issue by ornicar** [ddugovic/Stockfish#8]
A new version of stockfish is out! How difficult would it be to upgrade this fork? I honestly have no idea.

**Comment by ddugovic** [comment]
I think it shouldn't be difficult; I've been trying to reconstitute my development environment (I'm migrating and upgrading my development server) for the past month and I'm getting close to a point where I can do this!

I think it's just a matter of pulling/merging changes from the master repo; there's a merge conflict so I need to do the pull/merge on a development machine and then sync GitHub with that machine.  I've tried to do this pull/merge before and found command-line git confusing, but in theory it should be simple...

**Comment by apetresc** [comment]
Hey guys, I went ahead and did the merge on my fork.

There were some tricky parts to the merge conflicts, but in all cases, I looked at the `git blame` where @ddugovic had made the original change to figure out the intent, and did the sensible thing (I think).

Everything seems to work correctly, although I don't know how to test lichess KOTH-functionality specifically.

Hopefully this makes your jobs a bit easier :)
